doc/integrator/ngeo.rst.tmpl

```diff
@@ -11,16 +11,30 @@ The main page where we can redefine the header is in the file:
 ``geoportal/<package>_geoportal/static-ngeo/js/apps/Controller<interface>.js``
 where ``<interface>`` is the interface name.
 
-The style sheet file is ``geoportal/<package>_geoportal/static-ngeo/js/apps/sass/<interface>.scss``.
+In all mode:
```

Comment:
What is the all mode?

Reply:
simple and normal or complex (it hasn't a defined name...)

Reply:
how about "full application mode" (vs. "simple application mode")?

Reply:
Suggestion:
"In all application modes (simple and full):"